forms-shared/src/slovensko-sk/xmlBuilder.ts

```diff
@@ -1,15 +1,26 @@
 import { Builder } from 'xml2js'
+import { GenericObjectType } from '@rjsf/utils'
 
-const builderOptionsBase = {
-  xmldec: {
-    version: '1.0',
-    encoding: 'UTF-8',
+export const buildSlovenskoSkXml = (
+  xmlObject: GenericObjectType,
+  {
+    headless,
+    pretty,
+  }: {
+    headless: boolean
+    pretty: boolean
   },
```

Comment:
is this only for debug, or is any of it required in certain uses of the output ? Please detail in comments

Reply:
I understand headless (and that it'll be kind-of obvious from the usage), I wonder mainly whetter "pretty" must be false in certain cases (i.e. if signer is ok with prettified output, but when it goes into slovensko.sk it must be a single line etc)

if such cases do not exist consider it a moot point

if they exist, consider whether it should be commented / warned about here, or if the place where this is used in such way is the one where the comment should be left